### PR TITLE
RD-6402 Update Secrets Widget documentation

### PR DESCRIPTION
**Pull request by PatrycjaKubiczek**
RD-6402

- update Secrets screenshots
- update "working with Secrets"
- update Secrets Page

### CODE REVIEW

File: content/working_with/manager/using-secrets.md [hunk 1]
```diff
@@ -63,6 +63,17 @@ For more commands, see [secrets command line]({{< relref "cli/orch_cli/secrets.m
 
 ![Create Secret]( /images/manager/create_secret_dialog.png )
 
+### Creating a secret with Secret Provider from the {{< param cfy_console_name >}}
+1. Go to the [Secrets page]({{< relref "working_with/console/pages/secrets-page.md" >}}).
+2. Click **Create** in the [Secret Store Management widget]({{< relref "working_with/console/widgets/secrets.md" >}}).
+3. Insert the secret key
+4. Click on the checkbox "Retrieve the secret value from a secret provider"
```
Comment (qooban) [inline content/working_with/manager/using-secrets.md:70]: What do you think about adding something like (to be updated before committing as I didn't include correct link)?:

```suggestion
4. Click on the checkbox "Retrieve the secret value from a secret provider" (you must have at least one secret provider defined in {{< param product_name >}} to be able to use this option, see [Secret Providers widget](link to Secrets Provider widget) for details)
```

File: content/working_with/manager/using-secrets.md [hunk 2]
```diff
@@ -63,6 +63,17 @@ For more commands, see [secrets command line]({{< relref "cli/orch_cli/secrets.m
...
+5. Select the secret provider from the dropdown list
+6. Insert the secret path
```
Comment (qooban) [inline content/working_with/manager/using-secrets.md:72]: How about a bit more precise descripton:
```suggestion
6. Insert the secret's path in selected secret provider
```
?

File: content/working_with/manager/using-secrets.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ Secret `test` created
 
 For more commands, see [secrets command line]({{< relref "cli/orch_cli/secrets.md" >}}).
 
-### Creating a secret from the {{< param cfy_console_name >}}
+### Creating a secret with value from the {{< param cfy_console_name >}}
```
Comment (qooban) [inline content/working_with/manager/using-secrets.md:53]:
```suggestion
### Creating local secret from the {{< param cfy_console_name >}}
```

File: content/working_with/manager/using-secrets.md [hunk 4]
```diff
@@ -63,6 +63,17 @@ For more commands, see [secrets command line]({{< relref "cli/orch_cli/secrets.m
 
 ![Create Secret]( /images/manager/create_secret_dialog.png )
 
+### Creating a secret with Secret Provider from the {{< param cfy_console_name >}}
```
Comment (qooban) [inline content/working_with/manager/using-secrets.md:66]: Maybe to refer to terms convention used in this page better would be to have something like:

```suggestion
### Creating remotely provided secret from the {{< param cfy_console_name >}}
```

**Review by qooban — COMMENTED** [review]
Looks good in general.

Added some minor comments about being more precise or using consistent terminology.

I would also update [Secrets page](https://github.com/cloudify-cosmo/docs.getcloudify.org/blob/7353d858c042482bdfba1d112740e468ef8a3789/content/working_with/console/pages/secrets-page.md), OK?

**Comment by PatrycjaKubiczek** [comment]
@qooban updated Secret Page as well https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2524/commits/9cc7d67869f4c04f051e44f40dce0252028c58af ✔️

File: content/working_with/manager/using-secrets.md (hunk 1 above)
Comment (qooban) [reply content/working_with/manager/using-secrets.md]: As mentioned in the original comment - link to be updated 🙂

**Review by qooban — COMMENTED** [review]
Almost perfect 🙂 

Unresolved just one discussion and added a comment there.

File: content/working_with/manager/using-secrets.md (hunk 1 above)
Comment (PatrycjaKubiczek) [reply content/working_with/manager/using-secrets.md]: didn't notice that 🙈 https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2524/commits/ef89980c9af79fd8763678a8200af9e46696a842 updated link and one small typo :)

**Review by qooban — APPROVED** [review]

**Pull request merged**